app/Http/Controllers/AdminsController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use Hash;
+use Validator;
+
+class AdminsController extends Controller
+{
+
+    function NewAdmin (Request $r)
+    {
+    	$validation=Validator::make($r->all(),
+    	[
+    		"name"=>"required",
+    		"surname"=>"required",
+    		"email"=>"required|email|unique:users,email",
+    		"phone"=>"nullable|numeric",
+    		"password"=>"required|min:6",
+    		"config_password"=>"required|same:password",
+    		"is_active"=>"required|integer",
+    	]);
+
+    	if ($validation->fails()) 
+    	{
+
+    	return json_encode("Validation Error");
+
+    	} else {
+
+    	$user = new User();
+    	$user->name = $r->name;
+    	$user->surname = $r->surname;
+    	$user->email = $r->email;
+    	$user->phone = $r->phone;
+    	$user->phone = $r->phone;
+    	$user->type = 'admin';
+    	$user->is_active = $r->is_active;
+    	$user->password = Hash::make($r->password);
+    	$user->save();
+
+    	return json_encode("Success");
+
+    	}
+    }
+
+    function ShowAdmin(Request $r)
```

Comment:
It will be better to use the full word, rename the variable to $request

Reply:
The id should be received from the query, not from the request.
Check the project that I've sent you.